pkg/sfu/downtrack.go

```diff
@@ -762,16 +762,8 @@ func (d *DownTrack) WritePaddingRTP(bytesToSend int, paddingOnMute bool, forceMa
 		return 0
 	}
 
-	// LK-TODO Look at load balancing a la sfu.Receiver to spread across available CPUs
 	bytesSent := 0
 	for i := 0; i < len(snts); i++ {
-		// LK-TODO-START
-		// Hold sending padding packets till first RTCP-RR is received for this RTP stream.
-		// That is definitive proof that the remote side knows about this RTP stream.
-		// The packet count check at the beginning of this function gates sending padding
-		// on as yet unstarted streams which is a reasonable check.
-		// LK-TODO-END
-
```

Comment:
some drive-by clean up/fix up. We have the information about receiver reports in `RTPStats` module. So, using that to check whether to send padding.